src/clj/rems/css/styles.clj

```diff
@@ -243,15 +272,15 @@
    [:.text-highlight {:color (get-theme-attribute :color3)
                       :font-weight "bold"}]))
 
-(def ^:private dashed-form-group
+(defn- dashed-form-group []
```

Comment:
were you thinking of consistency with the `generate-foo` functions when you turned these from constants to functions? If so, should we use the `generate-` prefix for everything, or perhaps for nothing?

Reply:
I think it makes sense to use functions for these style fragments, because they might need to look up stuff from the config/theme. Mixing constants and functions can lead to surprises when developing, so maybe let's just make everything a function. At that point, I think the `generate-` prefixes can be dropped from the names, it's just noise.

Reply:
I just remember we did have a discussion with @Macroz the other day we decided to get rid of the ```private``` definitions

Reply:
I think this is a good point naming convention-wise. I can fix that @opqdonut

Reply:
Well, if we want to to drop it all together then we have to either separate then into mixins file and definitely do it in a standalone PR

Reply:
I think I remember the discussion. I was actually not suggesting then to replace these `def` with `defn`. There is no need to mark them as private here really. But also I was thinking about a case where one does `defn` and `^:private` which does not need to be so because `defn-` exists.